### ISSUE TITLE
Question task: potential bug with initial tab order & focus

### ISSUE THREAD

**Issue by snblickhan** [zooniverse/front-end-monorepo#5417]
## Package
_Choose from the list:_
- [ ] app-content-pages
- [ ] app-project
- [x] lib-classifier
- [ ] lib-panoptes-js
- [ ] lib-react-components
- [ ] unknown

## Is your feature request related to a problem? Please describe.
From a volunteer on the [Knitting Leaflets Project](https://www.zooniverse.org/projects/elliereed185/knitting-leaflet-project), referring to [this workflow](https://www.zooniverse.org/projects/elliereed185/knitting-leaflet-project/classify/workflow/24462).

> In the Basic workflow, the project settings for initial tab order & focus make data entry more difficult by requiring use of the mouse/touchpad.

> When a screen opens/appears, hitting Tab once should take me directly to the first place for adding information. Mostly, it doesn't. Here's what I found on each screen, in order:

> 1. Publisher name: Tab goes to the drop-down list of publisher names. (Yay!)
> 2. Leaflet number: The focus starts in the text entry box. (Yay!)
> 3. "Is there an image on this leaflet?": Tab goes to the Subject discussion icon.
> 4. Image or drawing? Subject discussion icon.
> 5. Drawing or photo? Subject discussion icon.
> 6. Contain color? Yes (yay!)
> 7. Item of technology? Subject discussion icon.
> 8. Person in image?Subject discussion icon.
> 9. Out-of-doors? Subject discussion icon.

> Whether you can fix that while the project is underway, I don't know. It would make life easier for some of us if you could.

## Describe the solution you'd like
It seems like this behavior should be consistent across tasks(?), but would welcome feedback on whether this is intentional so I can write back to this user and explain why this needs to be the case.

## Additional context
[Original Talk thread](https://frontend.preview.zooniverse.org/projects/elliereed185/knitting-leaflet-project/talk/4675/3112391?comment=5095306).

**Comment by eatyourgreens** [comment]
Seems pretty broken to me (Firefox 117 on MacOS 11.7.10) but all the broken tasks are question tasks, so it's a bug with that particular task type.

**Comment by eatyourgreens** [comment]
Also, is there a link for the Basic workflow?

**Comment by eatyourgreens** [comment]
The question is wrapped in a fieldset and legend (#5431.) One option would be to make the fieldset focusable, and send keyboard focus there as you work through the workflow tasks.

**Comment by eatyourgreens** [comment]
Something I did notice, working through that workflow on a reasonably wide screen: it's hard to scroll the subject image up and down while the task is autofocused on the right. Maybe check the box for those workflows to limit the subject height on wide screens?

**Comment by eatyourgreens** [comment]
Noting here that [keyboard access](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) is a level A WCAG requirement, currently failing for all workflows that use question tasks.
